lib/github_service/issue_mover.rb

```diff
@@ -0,0 +1,46 @@
+module GithubService
+  class IssueMover
+    Error = Class.new(StandardError)
+
+    attr_reader :old_issue, :new_issue, :fq_repo_name
+
+    def initialize(issue, fq_repo_name)
+      @old_issue = issue
+      @fq_repo_name = fq_repo_name
+      @organization_name, @repo_name = fq_repo_name.split("/")
+    end
+
+    def execute
+      validate
+
+      new_issue = GithubService.create_issue(fq_repo_name, old_issue.title, new_issue_body)
+      GithubService.add_comment(old_issue.fq_repo_name, old_issue.number, "I've moved this issue to #{new_issue.html_url}")
```

Comment:
"third person sans subject" should be "passive voice"....I don't know proper grammar terms 😛